charts/incubator/hyperswitch-helm/templates/secrets.yaml

```diff
@@ -6,17 +6,17 @@ data:
   ROUTER__ANALYTICS__SQLX__KMS_ENCRYPTED_PASSWORD: ""
   ROUTER__ANALYTICS__SQLX__USERNAME: ""
   ROUTER__API_KEYS__KMS_ENCRYPTED_HASH_KEY: ""
-  ROUTER__JWEKEY__LOCKER_DECRYPTION_KEY1: ""
-  ROUTER__JWEKEY__LOCKER_DECRYPTION_KEY2: ""
-  ROUTER__JWEKEY__LOCKER_ENCRYPTION_KEY1: ""
-  ROUTER__JWEKEY__LOCKER_ENCRYPTION_KEY2: ""
-  ROUTER__JWEKEY__LOCKER_KEY_IDENTIFIER1: ""
-  ROUTER__JWEKEY__LOCKER_KEY_IDENTIFIER2: ""
-  ROUTER__JWEKEY__TUNNEL_PRIVATE_KEY: ""
-  ROUTER__JWEKEY__VAULT_ENCRYPTION_KEY: ""
-  ROUTER__JWEKEY__VAULT_PRIVATE_KEY: ""
-  ROUTER__KMS__KEY_ID: ""
-  ROUTER__KMS__REGION: ""
+  ROUTER__JWEKEY__LOCKER_DECRYPTION_KEY1: {{ .Values.application.server.secrets.kms_jwekey_locker_decryption_key1 }}
```

Comment:
we are currently using no-kms image in CDK and helm. will this config affect it. If not we are good with the PR